site/ru/blog/new-in-devtools-98/index.md

```diff
@@ -6,228 +6,289 @@ authors:
 date: 2022-01-13
 updated: 2022-01-13
 description:
-  "Full-page Accessibility tree, more precise changes in the Changes tab, and more."
+  "Дерево специальных возможностей на всю страницу, явное отображение изменений на вкладке Изменения и многое другое."
 hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/Csl4vzzRO3Ei1ryeK4wc.jpg'
 alt: ''
 tags:
   - new-in-devtools
   - devtools
   - chrome-98
-draft: true
 ---
 
 *Переводы предоставлены [Alena Batitskaya](https://twitter.com/ABatickaya). Редактор — [Maxim Salnikov](https://twitter.com/webmaxru).*
 
 {% include 'partials/devtools/ru/banner.md' %}
 
 <!-- start: translation instructions -->
-<!-- 1. Remove the "draft: true" tag above when submitting PR -->
-<!-- 2. Provide translations under each of the English commented original content -->
-<!-- 3. Translate the "description" tag above -->
-<!-- 4. Translate all the <img> alt text -->
-<!-- 5. Update the whats-new.md file -->
+<!-- + 1. Remove the "draft: true" tag above when submitting PR -->
+<!-- + 2. Provide translations under each of the English commented original content -->
+<!-- + 3. Translate the "description" tag above -->
+<!-- + 4. Translate all the <img> alt text -->
+<!-- + 5. Update the whats-new.md file -->
 <!-- end: translation instructions -->
 
 <!-- ## Preview feature: Full-page accessibility tree {: #a11y-tree } -->
+## Ранний доступ: дерево специальных возможностей на всю страницу {: #a11y-tree }
 
 <!-- The new **Full-page accessibility tree** makes it easier for you to get an overview of the full-page [accessibility tree](/blog/full-accessibility-tree/#what-is-the-accessibility-tree) and help you better understand how your web content is exposed to assistive technology.  -->
+Новая функция **Дерево специальных возможностей на всю страницу** облегчает обзор полностраничного [дерева доступности](/blog/full-accessibility-tree/#what-is-the-accessibility-tree) и помогает лучше понять, как ваш веб-контент работает со вспомогательными технологиями. 
+
 
 <!-- In the **Elements** panel, open the **Accessibility** pane and select **Enable full-page accessibility tree** checkbox. Then, reload DevTools and you will see a new accessibility button in the **Elements** panel. -->
+В панели **Элементы** (Elements) откройте панель **Специальные возможности** (Accessibility) и поставьте галочку в чекбоксе **Дерево специальных возможностей на всю страницу** (Full-page accessibility tree). После этого перезагрузите DevTools и вы увидите новую кнопку в панели **Элементы** (Elements).
 
 <!-- Click on it to toggle to the **Full-page accessibility tree** view. You can expand nodes or click to see details in the  **Accessibility** pane. -->
+Нажмите на кнопку чтобы вкоючить режим **Дерево специальных возможностей на всю страницу**. Вы можете развернуть отдельные узлы или посмотреть подробноести в панели **Специальные возможности** (Accessibility).
 
```

Comment:
A passage is missing here:
Select a node and toggle back to the DOM tree view. The corresponding DOM node is selected now. This is a great way to understand the mapping between the DOM node and its accessibility tree node. This works for DOM tree ⬌ Accessibility tree view too!

Reply:
It's something new. I'll translate it right now =)